core/deployment/src/main/java/io/quarkus/deployment/steps/NativeImageFeatureStep.java

```diff
@@ -346,7 +346,7 @@ public void write(String s, byte[] bytes) {
             // Needed to access LOCALIZATION_FEATURE
             exports.produce(
                     new JPMSExportBuildItem("org.graalvm.nativeimage.builder", "com.oracle.svm.core.jdk.localization",
-                            GraalVM.Version.VERSION_22_1_0));
+                            GraalVM.Version.CURRENT));
```

Comment:
This should not change. It says that the export should be done for GraalVM >= 22.1